modules/oc-mirror-troubleshooting-v2.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-troubleshooting-v2_{context}"]
+== Troubleshooting oc-mirror plugin v2 Image Mirroring Errors
+
+oc-mirror plugin v2 now logs all image mirroring errors in a separate file, making it easier to track and diagnose failures.
+
+Additionally, when an image fails to mirror, oc-mirror plugin v2 informs the user which operator is incomplete as a result. This provides greater clarity and helps identify which operator bundles are affected by the mirroring errors.
+
+.Procedure
+
+. Rectifying server related issues:
+.. Open the errors.log file located in the oc-mirror output directory.
+.. Look for error messages like the one above, which typically indicate server-side issues (e.g., HTTP 500 errors, expired tokens, or timeouts).
+.. Retry the mirroring process or contact support if the issue persists.
+
+. Errors related incomplete Operators:
+.. Check for warnings in the console or log file indicating which Operators are incomplete.
+.. If an Operator is flagged as incomplete, the image related to that Operator likely failed to mirror.
+.. You might need to manually mirror the missing image or retry the mirroring process.
+
+
+. Errors related to generated cluster resources:   
+.. Even if some images fail to mirror, oc-mirror v2 will still generate cluster resources (IDMS.yaml and ITMS.yaml) for the successfully mirrored images.
+.. Check the output directory for the generated files.
+.. If these files are missing for specific images, ensure that no critical errors occurred for those images during the mirroring process.
+
+By following these steps, users can better diagnose issues and ensure smoother mirroring procedures.
```

Comment:
may be procedures is not required here. Just ending with mirroring is sufficient i believe.

Reply:
Done